src/main/java/duke/Ui.java

```diff
@@ -1,7 +1,5 @@
 package duke;
 
-import task.Task;
-
 import java.util.Scanner;
 
 public class Ui {
```

Comment:
The logo looks really impressive, and it's clear you put in a lot of effort. Great job, and keep up the good work! :fire::fire::fire: